15/index.php

```diff
@@ -189,15 +200,29 @@
 <h1 class='red underline'>Developer Software</h1>
 
 <?php
-downloadSection('Keyman Developer 15',    'developer', 'keymandeveloper-$version.exe', 'beta');
+downloadSection('Keyman Developer 15',    'developer', 'keymandeveloper-$version.exe', 'stable');
 ?>
 
 <h3>What's new in Keyman Developer 15?</h3>
 
+<h3>Breaking changes for keyboard developers</h3>
+
+<p>We work hard to minimize the potential for breaking changes to Keyman. We sometimes must make a change which may
+not be 100% backwardly compatible, either to correct a bug, or to address security issues. Breaking changes are changes
+which may prevent an existing keyboard from building without modification with the new compiler. The following issues are
+known breaking changes in Keyman 15.0 for keyboard developers:</p>
+
+<ul>
+  <li>Keyboard compiler now warns on inconsistent use of Caps Lock to prevent unexpected behavior of the keyboard in use (#6347)</li>
```

Comment:
```suggestion
  <li>Keyboard compiler now warns on inconsistent use of Caps Lock to prevent unexpected behavior of the keyboard in use -- [blog post](https://blog.keyman.com/2022/04/how-to-resolve-caps-and-ncaps-ambiguity-in-keyman-keyboards/) (#6347)</li>
```